### PR TITLE
Enhanced OpenID-Connect authentication via the CLI

### PR DESCRIPTION
**Pull request by abellotti**
Updating the OpenID-Connect authentication page to include OIDC via the enhanced appliance console CLI.

### CODE REVIEW

**Comment by abellotti** [comment]
keeping as WIP as code is not merged yet.

**Comment by abellotti** [comment]
Hi @jvlcek, updated the PR as discussed. 

image

**Comment by Fryguy** [comment]
@abellotti  Since you renamed the .conf file to .conf.erb, I think the manual instructions now need to reflect that change.  That is, does the user now have to copy from .conf to .conf.erb and modify the sections that look like `<% oidc... %>`

File: auth/openid_connect.adoc [hunk 1]
```diff
@@ -211,3 +216,74 @@ with enough administrative privileges to update it:
 * ssh to the appliance as root
 * Run _appliance_console_cli --extauth-opts_ *local_login_disabled=false*
 
+
+[[oidc-configuration-cli]]
+== Configuring OIDC using the CLI
+
+Another way to configure OpenID-Connect on the appliance is to use the appliance
+console CLI instead of the above steps. The `--oidc-config` subcommand of
+the  appliance console CLI allows one to configure the appliance for OpenID-Connect
```
Comment (Fryguy) [inline auth/openid_connect.adoc]: Extreme minor, but there are 2 spaces here.

File: auth/openid_connect.adoc [hunk 2]
```diff
@@ -211,3 +216,74 @@ with enough administrative privileges to update it:
...
+
+```
+--oidc-url=http://keycloak-server:8080/auth/realms/miq-realm/.well-known/openid-configuration
+```
+With *miq-realm* being the Keycloak Realm this OpenID-Connect
+appliance client is being configured for.
+
+The `--oidc-client-id` is the Client ID specified for
+the appliance in the OpenID-Connect provider, usually
+just `<miq-appliance-fqdn{gt}`
```
Comment (Fryguy) [inline auth/openid_connect.adoc]: `{gt}` ?

**Comment by abellotti** [comment]
good catch @Fryguy, I missed the update in the manual section, should be

```
cp ${TEMPLATE_DIR}/etc/httpd/conf.d/manageiq-external-auth-openidc.conf.erb \
    /etc/httpd/conf.d/manageiq-external-auth-openidc.conf
```

And then manually replacing the parameters.

File: auth/openid_connect.adoc (hunk 2 above)
Comment (abellotti) [reply auth/openid_connect.adoc]: yea, depending on where a > is sometimes rendered as >;  I will update all the ones that render properly as >.

**Pull request merged**